src/installer/tests/Microsoft.NET.HostModel.Tests/Helpers/BundleHelper.cs

```diff
@@ -90,11 +92,33 @@ public static string GetExtractionPath(TestProjectFixture fixture, Bundler bundl
             return Path.Combine(GetExtractionRootPath(fixture), GetAppBaseName(fixture), bundler.BundleManifest.BundleID);
 
         }
+
         public static DirectoryInfo GetExtractionDir(TestProjectFixture fixture, Bundler bundler)
         {
             return new DirectoryInfo(GetExtractionPath(fixture, bundler));
         }
 
+        public static OSPlatform GetTargetOS(string runtimeIdentifier)
+        {
+            return runtimeIdentifier.Split('-').First() switch {
+                "win" => OSPlatform.Windows,
+                "osx" => OSPlatform.OSX,
+                "linux" => OSPlatform.Linux,
+                _ => throw new ArgumentException(nameof(runtimeIdentifier))
+            };
+        }
+
+        public static Architecture GetTargetArch(string runtimeIdentifier)
+        {
+            return runtimeIdentifier.Split('-').Last() switch {
+                "x64" => Architecture.X64,
```

Comment:
This is basically wrong - for example `win81-arm64-aot` is a valid RID (see https://github.com/dotnet/runtime/blob/master/src/libraries/pkg/Microsoft.NETCore.Platforms/runtime.json and scroll to the end of the file).:
* Since this is test-only code - we should take the second part after splitting on `-`

Reply:
There's also `linux-musl-arm64` - I'll fix it to do what the SDK does.